content_script.js

```diff
@@ -14,8 +14,7 @@ class HTMLSerializer {
      *     ignored while serializing a document.
      * @const
      */
-     // TODO(sfine): Process links.
-    this.FILTERED_TAGS = new Set(['script', 'noscript', 'style', 'link']);
+    this.FILTERED_TAGS = new Set(['script', 'noscript', 'style']);
```

Comment:
No.  In general, for one line/small changes, how should I incorporate them into the pull request process?  Should I:
a) Create a separate pull request.
b) Lump them into another pull request.
c) Just commit them directly to master.

I suppose I just went with option b.  Should I undo it?